### PR TITLE
Simplified INSTALL.md to be in sync with README

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.
Ready for review and merge.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,112 +1,25 @@
...
-prerequisites. This approach is compatible with PyPI because `pip install`
-invokes `setup.py install` but not the new commands. It is also compatible
-with packaging PyWBEM into OS-level Python packages, for the same reason.
-This approach is also compatible with
-[virtual Python environments](https://docs.python-guide.org/en/latest/dev/virtualenvs/)
-because `sudo` is invoked under the covers for installing the OS-level
-packages, so you can still invoke setup.py without sudo for targeting the
-current virtual Python environment.
+  * Download the
+    [pywbem_os_setup.sh](https://pywbem.readthedocs.io/en/stable/_downloads/pywbem_os_setup.sh)
```
Comment (KSchopmeyer) [inline INSTALL.md]: Only covers linux and we really do need to cover windows.

Comment (andy-maier) [reply INSTALL.md]: The README also just covers Linux. This is likely to be the most common use case these days.

In both cases (README and INSTALL.md), we are referencing the install section and provide a link that takes you to exactly the install section.

Comment (KSchopmeyer) [reply INSTALL.md]: OK by me

**Comment by KSchopmeyer** [comment]
I am OK with it as it is. I had forgotten about this being standard on open source projects.  I do agree that we have too many different repetititons of the install info but that is not issue for this pr.

**Comment by KSchopmeyer** [comment]
Approved

**Pull request merged**